### PR TITLE
DDF-4235 Moving result form management out of workspace

### PR DESCRIPTION
**Pull request by andrewzimmer**
#### What does this PR do?
Moves result form management outside the workspace. As well as removing result forms as an experimental feature.
Search form and result form icons were also updated
#### Who is reviewing it? 
@Bdthomson @alanchampion @hayleynorton 

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining 
#### How should this be tested?
- Build/run
- Navigate to new result forms page
- Ensure no regression in result form functionality

#### What are the relevant tickets?
[DDF-4235](https://codice.atlassian.net/browse/DDF-4235)
#### Screenshots
resultform
screen shot 2018-10-29 at 2 04 05 pm
screen shot 2018-10-29 at 2 04 34 pm

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by Bdthomson** [comment]
In this PR can we change the icon for BOTH search forms and result forms now that they are separated?

**Comment by andrewzimmer** [comment]
Which icons would you suggest?

File: ui/packages/catalog-ui-search/src/main/webapp/component/navigator/navigator.hbs [hunk 1]
```diff
@@ -31,6 +31,9 @@
     <button class="navigation-choice is-neutral choice-search-forms is-button" data-fragment="forms">
         <span class="fa fa-file-o"></span><span>Search Forms</span><div class="forms-indicator"></div>
     </button>
+    <button class="navigation-choice is-neutral choice-result-forms is-button" data-fragment="resultForms">
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/component/navigator/navigator.hbs]: ❓ If we're naming this data-fragment `resultForms`, should we rename the one above `searchForms`?

File: ui/packages/catalog-ui-search/src/main/webapp/js/router/routes.js [hunk 2]
```diff
@@ -75,4 +75,12 @@
       classes: 'is-bold',
     },
   },
+  openResultForms: {
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/js/router/routes.js]: ✏️ same kind of thing here

**Comment by Bdthomson** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.less [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@{customElementNamespace}result-form-collection {
+  > .collection {
```
Comment (andrewkfiedler) [inline ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.less]: We'll need to have a display for `result-form-collection` set or we'll get some weird issues on certain browsers.  I would suggest `display: block`

**Comment by andrewzimmer** [comment]
build now

**Comment by Bdthomson** [comment]
build now (owasp download failure)

**Comment by hayleynorton** [comment]
🎉 Hero successful
- quick built
- verified that the Result Forms no longer appear in the search drop down menu
- verified that Search Forms and Result Forms have different icons
- verified that full Result Form functionality was maintained

**Pull request merged**